Files/View Models/ItemViewModel.cs

```diff
@@ -609,7 +609,7 @@ public async void LoadExtendedItemProperties(ListedItem item, uint thumbnailSize
                         {
                             matchingItem.FolderRelativeId = matchingStorageItem.FolderRelativeId;
                             matchingItem.ItemType = matchingStorageItem.DisplayType;
-                            using (var Thumbnail = await matchingStorageItem.GetThumbnailAsync(ThumbnailMode.SingleItem, thumbnailSize, ThumbnailOptions.ReturnOnlyIfCached))
+                            using (var Thumbnail = await matchingStorageItem.GetThumbnailAsync(ThumbnailMode.ListView, thumbnailSize, ThumbnailOptions.UseCurrentScale))
```

Comment:
We actually want to use `ThumbnailMode.SingleItem` here for more descriptive thumbnails.

Reply:
@Dylan-Osborne